src/test/resources/pypi-repo/pypi.yml

```diff
@@ -0,0 +1,9 @@
+---
+repo:
+  type: pypi
+  storage:
+    type: fs
+    path: /var/artipie/data/
+  settings:
```

Comment:
@milkysoft this section is not required for pypi repo

Reply:
@olenagerasimova , removed with latest commit